File: Components/Reporting/VB/DevExpress.DemoReports/Data/Employee.vb
Imports System
Imports System.Collections.Generic

Namespace DevExpress.DemoData.Models
    Partial Public Class Employee
        Public Property EmployeeID() As Long
        Public Property LastName() As String
        Public Property FirstName() As String
        Public Property Title() As String
        Public Property TitleOfCourtesy() As String
        Public Property BirthDate() As Date?
        Public Property HireDate() As Date?
        Public Property Address() As String
        Public Property City() As String
        Public Property Region() As String
        Public Property PostalCode() As String
        Public Property Country() As String
        Public Property HomePhone() As String
        Public Property Extension() As String
        Public Property Photo() As Byte()
        Public Property Notes() As String
        Public Property ReportsTo() As Long?
        Public Property Email() As String

        Public ReadOnly Property PageHeader() As String
            Get
                Return (FirstName & " " & LastName).ToUpper()
            End Get
        End Property
        Public ReadOnly Property PageContent() As String
            Get
                Return FirstName & " " & LastName & " was born on " & DateToString(BirthDate) & ". Now lives at " & City & ", " & Country & ". " & TitleOfCourtesy & " " & LastName & " holds a position of " & Title & " our " & Region & " deparment, (" & City & " " & Country & "). Joined our company on " & DateToString(HireDate) & "."
            End Get
        End Property
        Private Function DateToString(ByVal [date]? As Date) As String
            If [date] Is Nothing Then
                Return Nothing
            End If
            Dim Months() As String = { "January", "February", "Marth", "April", "May", "June", "July", "August", "September", "October", "November", "December" }
            Return [date].Value.Day & "th of " & Months([date].Value.Month - 1) & " in " & [date].Value.Year
        End Function

    End Class
End Namespace

File: Components/Reporting/CS/DevExpress.DemoReports/Data/Employee.cs
using System;
using System.Collections.Generic;

namespace DevExpress.DemoData.Models
{
    public partial class Employee
    {
        public long EmployeeID { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string Title { get; set; }
        public string TitleOfCourtesy { get; set; }
        public Nullable<System.DateTime> BirthDate { get; set; }
        public Nullable<System.DateTime> HireDate { get; set; }
        public string Address { get; set; }
        public string City { get; set; }
        public string Region { get; set; }
        public string PostalCode { get; set; }
        public string Country { get; set; }
        public string HomePhone { get; set; }
        public string Extension { get; set; }
        public byte[] Photo { get; set; }
        public string Notes { get; set; }
        public Nullable<long> ReportsTo { get; set; }
        public string Email { get; set; }

        public string PageHeader { get { return (FirstName + " " + LastName).ToUpper(); } }
        public string PageContent {
            get {
                return FirstName + " " + LastName + " was born on " + DateToString(BirthDate) + ". Now lives at " +
                    City + ", " + Country + ". " + TitleOfCourtesy + " " + LastName + " holds a position of " + Title + " our " +
                    Region + " deparment, (" + City + " " + Country + "). Joined our company on " + DateToString(HireDate) + ".";
            }
        }
        string DateToString(DateTime? date) {
            if(date == null)
                return null;
            string[] Months = { "January", "February", "Marth", "April", "May", "June", "July", "August", "September", "October", "November", "December" };
            return date.Value.Day + "th of " + Months[date.Value.Month - 1] + " in " + date.Value.Year;
        }

    }
}
